test/unit/test_fracture_importer.py

```diff
@@ -251,5 +251,95 @@ def test_create_fracture(self):
         self.assertTrue(f.p[2].min() == 0)
 
 
+class TestImportDFN(unittest.TestCase):
```

Comment:
Assumption on ambient dimension? Should be reflected in the class and function name.